RELEASE-NOTES.md

```diff
@@ -1,5 +1,171 @@
+- Replace our customized `Duration` types with plain `timedelta`s — https://github.com/PrefectHQ/prefect/pull/13603
+- Remove dependencies from previously vendored starlette — https://github.com/PrefectHQ/prefect/pull/13600
+- Remove old task runners and futures modules — https://github.com/PrefectHQ/prefect/pull/13593
+- Correcting API schemas to maintain compatibility with Cloud — https://github.com/PrefectHQ/prefect/pull/13623
+- Add _sync flag to sync compatible fns — https://github.com/PrefectHQ/prefect/pull/13627
+- Bumping compatibility tests to latest on main — https://github.com/PrefectHQ/prefect/pull/13635
+- Update `apply_async` interface to mirror `multiprocessing` equivalent — https://github.com/PrefectHQ/prefect/pull/13643
+- Bringing back the integration tests — https://github.com/PrefectHQ/prefect/pull/13645
+- Update task server to run received tasks concurrently — https://github.com/PrefectHQ/prefect/pull/13653
+- Disable the automations-assessment test for Prefect Cloud — https://github.com/PrefectHQ/prefect/pull/13659
+- migrate `prefect-github` to pydantic v2 — https://github.com/PrefectHQ/prefect/pull/13655
+- migrate `prefect-gitlab` to pydantic v2 — https://github.com/PrefectHQ/prefect/pull/13656
+- Bumping compat-tests — https://github.com/PrefectHQ/prefect/pull/13671
+- Add a Join the Community CTA — https://github.com/PrefectHQ/prefect/pull/13668
+- migrate `prefect-email` to pydantic2 — https://github.com/PrefectHQ/prefect/pull/13654
+- migrate `prefect-slack` to pydantic 2 — https://github.com/PrefectHQ/prefect/pull/13673
+- Allows us to run integration package unit tests by adding a PR label — https://github.com/PrefectHQ/prefect/pull/13665
+- Adjustments to make sure that the client build still works — https://github.com/PrefectHQ/prefect/pull/13682
+- Add ability to limit concurrent runs for a task server — https://github.com/PrefectHQ/prefect/pull/13662
+- Mark tasks as being `deferred` when using `apply_async` — https://github.com/PrefectHQ/prefect/pull/13681
+- migrate `prefect-gcp` to pydantic 2 — https://github.com/PrefectHQ/prefect/pull/13650
+- removing references to deprecated block types, adding disclaimer — https://github.com/PrefectHQ/prefect/pull/13651
+- migrate `prefect-shell` to pydantic 2 — https://github.com/PrefectHQ/prefect/pull/13675
+- Add a naive distributed future — https://github.com/PrefectHQ/prefect/pull/13611
+- Small Transaction refactor — https://github.com/PrefectHQ/prefect/pull/13692
+- adding in empty state import — https://github.com/PrefectHQ/prefect/pull/13696
+- Add Task.delay() — https://github.com/PrefectHQ/prefect/pull/13695
+- Simplify rate limits page — https://github.com/PrefectHQ/prefect/pull/13689
+
+## New Contributors
+* @jaraics made their first contribution in https://github.com/PrefectHQ/prefect/pull/13144
+* @mitchnielsen made their first contribution in https://github.com/PrefectHQ/prefect/pull/13562
```

Comment:
```suggestion
```

Reply:
we can probably just have one section for this, when we have both new and regular contributors